student.rb

```diff
@@ -1,9 +1,10 @@
-require_relative 'person'
+require './person'
+
 class Student < Person
-  attr_accessor :classroom
+  attr_reader :classroom, :parent_permission, :name, :age
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  def initialize(age, name, classroom, parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ]  In the Student class, there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). Check out [this article with examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more information.